build/Uno.WinUI.MediaPlayer.Skia.Gtk.nuspec

```diff
@@ -0,0 +1,24 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+	<metadata minClientVersion="5.0">
+		<id>Uno.UI.MediaPlayer.Skia.Gtk</id>
+		<version>0.0.1</version>
+		<authors>nventive</authors>
+		<icon>uno.png</icon>
+		<projectUrl>https://github.com/unoplatform/uno</projectUrl>
+		<license type="expression">Apache-2.0</license>
+		<description>This package provides support for MediaPlayerElement for Uno Platform Skia+Gtk apps</description>
+		<copyright>nventive</copyright>
```

Comment:
Should this be uno platform rather than nventive?

Reply:
Not for this release, it's changed in breaking changes, we'll do another round on master merge.